java/arcs/core/storage/driver/volatiles/VolatileMemory.kt

```diff
@@ -37,6 +37,9 @@ interface VolatileMemory {
     updater: (VolatileEntry<Data>?) -> VolatileEntry<Data>
   ): Pair<Boolean, VolatileEntry<Data>>
 
+  /** Get the number of elements in volatile memory storage. */
+  fun count(): Long
```

Comment:
maybe countEntries? this counts all entries, so both entities as well as containers (singletons/collections).
so it does not seem getEntitiesCount is counting just entities. eg imagine you store a collection with two entities, you will have 3 volatileEntry right?

Reply:
Yeah, this isn't totally satisfactory. Any idea about how we could filter volatile entries to get just the entities?

Reply:
One option is to have this count only entities (and call it countEntities): the way to do that is to inspect each VolatileEntry to see if it contains a CrdtSet or CrdtEntity and only count crdtentities.
The other option is to have this count all entries (and call it countEntries): then modify you test accordingly. Eg if you expect 2 entities in 1 collection, you would assert 3 entries in the test. But in this case it should not be wired through the getEntitiesCount abstract method.